proto/profile.proto

```diff
@@ -134,6 +169,15 @@ message Label {
   int64 num_unit = 4;  // Index into string table
 }
 
+// A group of labels that can be referenced multiple times in order to reduce
+// overhead. Example use cases for reusable label sets:
+//
+// - Tracing IDs (e.g. trace_id=123, span_id=345)
+// - Thread/Process IDs (e.g. process_id=123, thread_id=456)
+message LabelSet {
+  repeated Label label = 1;
+}
```

Comment:
```suggestion
message LabelSet {
  uint64 id = 1;
  repeated Label label = 2;
}
```

This would be more consistent. I don't know why pprof allows for id schemes other than `offset + 1` 🤷🏻  so I'm not sure if it's a good idea or not, just pointing it out.

Reply:
Great catch, fixed in dd8e60a6c8e25092773eea72292cc2f2d18c04e9 .

> Similarly, a LabelSet.id of id should probably be forbidden?

You mean it should be forbidden for a LabelSet to contain a Label with key="id" ? Not sure I'm following.

Reply:
Ha, was just a mistake. I meant "`Label.id` of zero".

Reply:
I think the absence of label set ids is represented as the absence of a `Breakdown.label_set_id` field, which a decoder will probably represent as an empty list of label set ids?

dd8e60a6c8e25092773eea72292cc2f2d18c04e9 already adds a comment that says that `Label.id` must not be zero.